fluidkeys/main.go

```diff
@@ -63,6 +63,7 @@ Configuration file: %s
 Usage:
 	fk secret send <recipient-email-address>
 	fk secret receive
+	fk setup
```

Comment:
seems like this should be at the top, shouldn't it?

Reply:
Yep!